templates/auth0-sso-handler-lock10.php

```diff
@@ -13,19 +13,20 @@
   });
 
   var options = <?php echo json_encode( $lock_options->get_sso_options() ); ?>;
+  options.responseType = 'token id_token';
   webAuth.checkSession(options
   , function (err, authResult) {
       if (typeof(authResult) === 'undefined') {
         return;
       }
 
       if (typeof(authResult.code) !== 'undefined') {
```

Comment:
will this check fail if `authResult.code` is `null`?

Reply:
Potentially but there is an `idToken` fallback:

https://github.com/auth0/wp-auth0/blob/master/templates/auth0-sso-handler-lock10.php#L22

Reply:
And what if none of the `if` branches are fulfilled ? Shouldn't you handle this gracefully?

Reply:
Probably? But this is trying to auto-login with SSO so falling through = login form. Technically graceful? :)